tests/gui/measure/_defaults/test_aliases.py

```diff
@@ -0,0 +1,106 @@
+
+    def test_no_capital_letters_in_options_and_values(self, alias_config):
+        """Ensure all options and non-parent_aliases values are lowercase."""
+        for section in alias_config.sections():
+            for option in alias_config.options(section):
+                assert option == option.lower(), (
+                    f'Option "{option}" in section [{section}] '
+                    'contains uppercase letters.'
+                    )
+                if option == 'parent_aliases':
+                    # Skip class names as these contain uppercase strings.
+                    continue
+                if len(option.split('/')) == 3:
+                    # The option contains an overwrite value
+                    # that may contain capitalized letters.
+                    continue
+                if option != 'fallback_values':
+                    value = alias_config[section][option]
+                    assert value == value.lower(), (
+                        f'Alias "{value}" in [{section}][{option}] '
+                        'contains uppercase letters.'
+                        )
+                    continue
+                # Treat key 'fallback_values' differently because the
+                # fallback values may contain capitalized letters.
+                for value in ast.literal_eval(alias_config[section][option]):
+                    assert value[0] == value[0].lower(), (
+                        f'Alias "{value[0]}" in [{section}][{option}] '
+                        'contains uppercase letters.'
+                        )
+
+    def test_no_child_parent_option_overlap(self, alias_config):
+        """Ensure that child sections do not redefine parent_aliases."""
+        for section in alias_config.sections():
+            parents_raw = alias_config.get(section, 'parent_aliases',
+                                           fallback=None)
+            if not parents_raw:
+                continue
+            parents = ast.literal_eval(parents_raw)
+            # Collect all option names from parents.
+            parent_options = set()
+            for parent in parents:
+                parent_options.update(alias_config.options(parent))
+
+            # Check overlap between child options and parent options.
+            child_options = set(alias_config.options(section))
+            overlap = parent_options & child_options
+
+            assert not overlap, (
+                f'Section [{section}] redefines option(s) already '
+                f'defined in its parent_aliases {parents}: {overlap}'
+                )
+
+    def test_no_multiple_inheritance(self, alias_config):
+        """Check that there is no multiple inheritance."""
+        # Collect all sections that declare parent_aliases themselves.
+        inheriting_sections = (
+            section
+            for section in alias_config.sections()
+            if alias_config.has_option(section, 'parent_aliases')
+            )
```

Comment:
Watch out, this is a generator that will be exhausted at the first iteration. Use a set instead:
```suggestion
        inheriting_sections = {
            section
            for section in alias_config.sections()
            if alias_config.has_option(section, 'parent_aliases')
            }
```